ledger/appcow.go

```diff
@@ -425,6 +425,17 @@ func (cb *roundCowState) DelKey(addr basics.Address, aidx basics.AppIndex, globa
 	return nil // note: deletion cannot cause us to violate maxCount
 }
 
+// AppendLog adds message in logs
+func (cb *roundCowState) AppendLog(idx uint64, value string) error {
+	cb.logs = append(cb.logs, basics.LogItem{ID: idx, Message: value})
+	return nil
+}
+
+// GetLogs returns logs
+func (cb *roundCowState) GetLogs() []basics.LogItem {
```

Comment:
is it really used somewhere except tests? I do not think a public method needed only for test usage